drivers/sx127x/sx127x_netdev.c

```diff
@@ -343,6 +342,16 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             *((netopt_enable_t*) val) = sx127x_get_rx_single(dev) ? NETOPT_ENABLE : NETOPT_DISABLE;
             break;
 
+        case NETOPT_TX_POWER:
+            assert(max_len >= sizeof(int8_t));
+            *((int8_t*) val) = sx127x_get_tx_power(dev);
+            return sizeof(int8_t);
+
+        case NETOPT_IQ_INVERT:
+            assert(max_len >= sizeof(uint8_t));
+            *((netopt_enable_t*) val) = sx127x_get_iq_invert(dev) ? NETOPT_ENABLE : NETOPT_DISABLE;
```

Comment:
Can you split to fit within 80 characters per line please ?

Reply:
ok ! (other lines in this file would need to be split  as well)